code/index.html

```diff
@@ -20,25 +20,38 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the character have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
```

Comment:
An option here would be to add one more general option here, such as "Select an option below" and make it selected disabled hidden, to make sure also brown hair can be selected. The new option would then only show up at first view, before something has been selected.